geonode/qgis_server/helpers.py

```diff
@@ -82,3 +89,138 @@ def validate_django_settings():
             "{package}.".format(package=qgis_server.BACKEND_PACKAGE))
 
     return True
+
+
+def map_thumbnail_url(instance):
```

Comment:
Check this out @Gustry , what I did is to put this as a helper function and not a django view. Then I deleted map_thumbnail django view. Is it ok to do this?